### PR TITLE
Moths, except I have stolen them and they no longer conflict

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan and More Robust Than You
add: Adds moths! They're a new species that is able to fly in pressurized areas provided gravity is off... and that they still have wings.
/:cl:

**CREDITS TO @tortellinitony, @FuryMcFlurry, and @TDcoolguy300 for their original work on moths.**

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -393,6 +393,7 @@ ROUNDSTART_RACES human
 ## Races that are strictly worse than humans that could probably be turned on without balance concerns
 ROUNDSTART_RACES lizard
 #ROUNDSTART_RACES fly
+#ROUNDSTART_RACES moth
```
Comment (Xhuis) [inline config/game_options.txt]: Now just remove that `#` and we'll be merge-ready

Comment (ChangelingRain) [reply config/game_options.txt]: Shoo, shoo!

**Comment by Buggy123** [comment]
https://imgur.com/4UQ4jNY.jpg

**Comment by PKPenguin321** [comment]
Their back sprite looks like a slice of pizza

**Comment by Repukan** [comment]
*scream

**Comment by awesine** [comment]
*scream merge now

**Comment by Armhulen** [comment]
gettin real sick and tired of all this pr boogaloo

**Comment by CrackParrot** [comment]
Add the scream merge so i can have a reason to kidnap and remove their tongue in-game

**Comment by Spaceinaba** [comment]
why are you guys even asking for the scream to be added it has already been removed multiple times

**Comment by Buggy123** [comment]
*scream

**Comment by ACCount12** [comment]
What has happened to all the people who were seemingly opposed to fetish content being added?

**Comment by MoreRobustThanYou** [comment]
they got impaled with spikes

**Comment by MoreRobustThanYou** [comment]
i'm just gonna post cute moths every day til this is merged....

and if it isn't? it'd hurt, alright
# FOR YOU.

**Comment by CrackParrot** [comment]
Is this shit ready to be merged or not?

File: config/game_options.txt (hunk 1 above)
Comment (Cyberboss) [reply config/game_options.txt]: >People still debating codebase config

I should remove the whole thing

**Comment by FantasticFwoosh** [comment]
> What has happened to all the people who were seemingly opposed to fetish content being added?

They're here but they know their comments would end up being deleted.

**Comment by CrackParrot** [comment]
> They're here but they know their comments would end up being deleted and too tired to post about this drawn out subject.

after seeing a few "special" cases of people taking the moth meme in-game too seriously i can understand why.

**Comment by FantasticFwoosh** [comment]
That wouldn't have anything to do [with something this](https://www.reddit.com/r/SS13/comments/7qn5g2/captain_comments_on_tg_having_mothmen/) would it @CrackParrot ? Or do you mean protestors in OOC?

**Comment by CrackParrot** [comment]
> That wouldn't have anything to do with something this would it @CrackParrot ? Or do you mean protestors in OOC? Forgive me for asking but you're being awfully vague.

It has everything to do with what you just said. i recall at one point i was playing HOP during the test merge and i swear to Christ there was like half the station full of those things. Eventually one of the "special" cases came up to me and asked for all access and i said no, so he wailed REEEE!!! at the top of his lungs and began vandalizing my office and all the other moths in the area dog-piled me i swear too god this had happened.

**Comment by Potato-Masher** [comment]
Despite what they say, mothtide is a thing.

In reference to assaulting the HoP for not giving access, why has it recently become TG culture to aquire all-access every god damn round? It's really starting to piss me off due to a select few individuals.

**Comment by McBawbaggings** [comment]
@Potato-Masher 
The moth metagang was an actual issue on hippie.
Any snowflake thing will encourage that behavior though, it's not exclusive to moths.

**Comment by PaznerIV** [comment]
Fun fact moths do not have mouths and any mouth you see on them is nonfunctional, and is just a left over from when they were in there larvae stage

**Comment by Xhuis** [comment]
how do they eat?!

**Comment by Hatfish** [comment]
They subsist entirely on light, fabric, and snuggles.

**Comment by PaznerIV** [comment]
They eat everything at their larvae stage and there life span does change depending on how much they eat during that stage.

**Comment by Supermichael777** [comment]
That is species dependent, some moths and butterflies are very short lived.

**Comment by Buggy123** [comment]
Can we still get the scream but if you do it more than a couple times in a period of time you scream yourself hoarse and go permanently mute?

**Comment by ExcessiveUseOfCobblestone** [comment]
> Fun fact moths do not have mouths and any mouth you see on them is nonfunctional, and is just a left over from when they were in there larvae stage

MAKE MOTHS PROFESSIONAL MIMES IN LORE

**Comment by 81Denton** [comment]
>In reference to assaulting the HoP for not giving access, why has it recently become TG culture to aquire all-access every god damn round?

You start with an egun, flash and a baton m8

**Comment by Potato-Masher** [comment]
@81Denton 
Those things might be nice against like one or two greyshits, but when 3-6 of them are clamoring for your access, then you most likely are going to lose due to disarm RNJesus.

**Comment by ACCount12** [comment]
This is getting more cancerous than furries on para.

**Comment by CrackParrot** [comment]
Wow

**Comment by Slignerd** [comment]
bombyxmoth7_small

**Comment by MrStonedOne** [comment]
Well we got to beat microsoft this year on comments some how.

**Comment by Slignerd** [comment]
@ACCount12 You haven't seen much of para, have you?

**Comment by FuryMcFlurry** [comment]
I didn't imagine my creation/shitpost to reach such a critical mass

**Comment by Grazyn** [comment]
Wait, isn't there a rule that a roundstart race can only be added if another roundstart race is disabled? So what is it gonna be?

**Comment by Slignerd** [comment]
@Grazyn Is there? I don't remember such rule being in places when flies were made roundstart.

And ironically enough, my vote would be on flies.

**Comment by CrackParrot** [comment]
> Is there? I don't remember such rule being in place when flies were made roundstart.
And ironically enough, if we actually had to do it, my vote would be on flies.

You sure about that? what about them Cat-People? its a hard decision.

**Comment by MoreRobustThanYou** [comment]
yeah let's disable cat degenerates

**Comment by Buggy123** [comment]
Okay, so I feel like I should clear this up. I _do not_ like furries. Its not because they want to be a anthromorphic cat who fucks other cats or someshit, fine, whatever, you do you; it's no worse than scat fetishists. No, my issue is that they love to shove it in other people's faces. They have this sort of _thing_ where their own self-image is tied to this, it isn't just some weird fetish.

And I still wouldn't care about things like this except that some of them insist that other people recognize this fact and go along with their impossible delusion. I do _not_ like it when people shove sexuality and fetishism into other peoples faces.

 And I also am a very... extreme person, I think that if you do something you should do it at least 100%, and if you can't even do 100% you shouldn't do it at all. And furries can't possibly ever be what they _want_ to be, give it a few hundred years and maybe improving technology would let them but they certainly can't _now_. And they do weird shit like fursuits and stuff where you're willfully doing like 10% and nothing more and just... ugh.


But I also like to be informed, and while I've heard of plenty of cat and dog fetish stuff I've certainly never heard of moths or even insects in general. So, I came to a decision. I donned a hazmat suit, booted up Malwarebytes, and went somewhere, a place I shall not name for your own sanity, to figure out just how much moths are fetishized.

In this dreadful place full of dreadful images, I searched for "cat" "lizard" "fly" "moth" and "skeleton", obviously corresponding to catpeople, liggers, flypeople, moths, and plasmamen.

image
image
image
image
image

I assume that the top number is the most accurate figure.

As you can see, cats have a astonishing large presence, over seven times larger than even lizards. Flypeople have a relatively small presence, at least compared to cats, and moths are even smaller. Skeletons... no idea what's going on there.

Next, I repeated this but limited my results to "mature" and "adult" content. I also carefully covered the bottom half of my screen where the results are visible, in order to preserve my dwindling sanity.

image
image
image
image
image

Finally, I limited my search to "adult" content and did it all again.

image
image
image
image
image

So, we end up with these figures:

Cats: 791019 total, 26% sexualized, 15% really sexual, 78% of total.
Lizards: 115319 total, 43% sexualized, 28% really sexual, 11% of total.
Flies: 61039 total, 21% sexualized, 11% really sexual, 6% of total.
Moths: 20371 total, 20% sexualized, 10% really sexual, 2% of total.
Skeletons: 22792 total, 18% sexualized, 7% really sexual, 2% of total.

For any particular furry out of these 5:

58% chance they "only" like cats
9% chance they think cats are hot
11% chance they fuck cats
6% chance they "only" like lizards
2% chance they think lizards are hot
3% chance they fuck lizards
5% chance they "only" like flies
1% chance they think flies are hot
0.7% chance they fuck flies
2% chance they "only" like moths
0.2% chance they think moths are hot
0.2% chance they fuck moths
2% chance they "only "like skeletons
0.2% chance they think skeletons are hot
0.2% chance they fuck skeletons

I have no idea how to tell how many furries are on tgstation, so lets go with a worse-case scenario: everyone on tgstation is a furry.

The website I got this data from has a total of 11 million submissions as of 2015, and added 3 million in 2016. I'll extrapolate and assume they're at 17 million by now. Thus, all of these categories combined are approximately 6% of the total furry presence.

If we assume tgstation has 1000 total frequent users, and thus 1000 furries, 60 of those are in one of these categories.

47 cat-ers
7 lizard-ers
4 fly-ers
1 moth-er
1 skeleton-er

So, if we just find that one guy who's a moth furry and ban him, we're all good!

Conclusion:
My methodology is far from perfect, but I have a mighty thirst that only bleach can fill, so if you want better results do it yourself.

**Comment by Repukan** [comment]
Had a good tug there huh

**Comment by Buggy123** [comment]
I don't want to have eyeballs anymore

**Comment by Slignerd** [comment]
lmao what the fuck

**Comment by RemieRichards** [comment]
This went places.

**Comment by Buggy123** [comment]
tl;dr moths are a-okay in my book because worst case scenario there's like 1 guy who actually gets off from it and everyone else just wants to *scream meme.

**Comment by Slignerd** [comment]
>9% chance they think cats are hot
>11% chance they fuck cats
You think 2% people fuck cats even though they don't find them hot?

**Comment by The-clown-known-as-Pepe** [comment]
@Buggy123 very nice overall but you used the wrong category for catpeople which should be instead cat_humanoids

**Comment by Buggy123** [comment]
Sorry for not looking around really thoroughly and getting a better idea of what search terms to use but practically speaking I couldn't finish it if I did because at some point the urge to gouge my eyes out would win.

**Comment by FantasticFwoosh** [comment]
I have no mouth and i must *scream.

**Comment by The-clown-known-as-Pepe** [comment]
@Buggy123 you must finish your research

**Comment by Buggy123** [comment]
no.

**Comment by Buggy123** [comment]
I can rub my eyes all I want but I cannot unsee what has been seen.

**Comment by FantasticFwoosh** [comment]
@The-clown-known-as-Pepe stop pressurising them into it, slither away back to moffcord whence you came, begone!

## **✝️**

**Comment by Buggy123** [comment]
MOTH IS ANGERY THAT THIS STILL ISN'T MERGED

**Comment by kyrahabattoir** [comment]
@Buggy123 Can you stop being a constantly offended little virgin?

**Comment by Qbopper** [comment]
@buggy123 does your really epic and cool post add anything to the pr

**Comment by Buggy123** [comment]
I mean, hopefully it demonstrates that furries aren't actually a issue here.

**Comment by RogueSteampunker** [comment]
What a fun ride. I loved the part where the discussion to a 180, snapping my neck instantly via whiplash

Moffs, everyone. Remove the players, not the innocents night butterflies

**Comment by Errorin** [comment]
So despite the fact you claim to not like furries, your later and previous actions betray your true allegiance, including your blatant  support of moths and furry races.

Nice try, spy guy.

File: config/game_options.txt (hunk 1 above)
Comment (praisenarsie) [reply config/game_options.txt]: i should remove you

**Comment by Buggy123** [comment]
Fuck you caught me, I'm a hardcore furry because I'm literally only okay with moths and flypeople.  Yiff me.

**Comment by KorPhaeron** [comment]
Can we nominate individual PR comments for fuckup of the year?

**Comment by praisenarsie** [comment]
you wish, kor

**Comment by Buggy123** [comment]
I just think moths are cute as fuck and *scream was hilarious (if occasionally overdone) is that a crime?

**Comment by awesine** [comment]
hi furry here merge this now

**Comment by CrackParrot** [comment]
I want to die now

**Comment by Qbopper** [comment]
@Buggy123 i'm on the side of "the furry argument is retarded" and your comment was still unbelievably dumb

**Comment by sawrge** [comment]
@buggy123 is definitely a furfag. No one goes that far to prove they aren't something unless they actually are, and are just really, _really_ insecure about it.

**Comment by scrubs2009** [comment]
Moths are unique as opposed to your boring generic ligger/furry. I like em. Plus the different wing types are neat. And of course *scream.

The metagangs will go away as people get used to moths and they're no longer a novelty.

**Comment by Buggy123** [comment]
>tried to make a post giving somewhat well researched evidence that mothfurries aren't really a issue

>everyone thinks he's a furry because of it

**Comment by Hatfish** [comment]
merge it or close it quick so i never have to look at the comments section in this pr again please

**Comment by Slignerd** [comment]
Wait, your argument like "furries aren't the issue here"?

It somehow read as the exact opposite.

**Comment by ma44** [comment]
>they don't move near light

**Comment by Buggy123** [comment]
Yes, my stance was "furries aren't a issue here cause no one wants to fuck moths". Turns out, I didn't quite plan out the part where I go on a furry image board to search for furry images and then post proof that I did so, without convincing everyone i'm a furry.

**Comment by sawrge** [comment]
@buggy123 fuck off furfag, yiff in hell.

**Comment by Slignerd** [comment]
I mean, you want on an umprompted rant about furry fetishists in moth PR, so at first I thought you were both anti-furry and anti-moth.

**Comment by kevinz000** [comment]
Why do you respond to buggy

**Comment by Buggy123** [comment]
I was thinking that half the reason no one wants to add new races is because most of them would attract furries, and more furries=more species=a slow transformation into Vorestation. I was making a point in moth's favor by demonstrating that most furries don't want to fuck moths.

**Comment by FuryMcFlurry** [comment]
make them see light as darkness and darkness as light.

**Comment by OneOneThreeEight** [comment]
that comment was well-researched and very insightful but I mean *what is it even for*

jesus. im sure this moth PR would've done well  without it

**Comment by optimumtact** [comment]
Here I go do deleting again

**Comment by optimumtact** [comment]
Did that all manually
